src/installer/pkg/packaging/rpm/pack-deps.proj

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project='package.props' />
```

Comment:
Double quotes are conventional

```suggestion
  <Import Project="package.props" />
```

Reply:
I've only seen usage of single quotes in related source files like: https://github.com/dotnet/runtime/blob/master/src/installer/pkg/packaging/rpm/package.targets

Reply:
Interesting, never noticed. I guess at the end of the day it's just XML... fixing over to convention would be nice though.

Reply:
Removing this whole proj file.